### PR TITLE
Fix/npm build

### LINKED ISSUE #123: Can't use in vite + vue3
Could not resolve "../build/annotation"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:5:33:
      5 │ import { ULabelAnnotation } from '../build/annotation';
        ╵                                  ~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/subtask"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:6:30:
      6 │ import { ULabelSubtask } from '../build/subtask';
        ╵                               ~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/geometric_utils"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:7:31:
      7 │ import { GeometricUtils } from '../build/geometric_utils';
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/annotation_operators"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:8:42:
      8 │ import { get_annotation_confidence } from '../build/annotation_operators';
        ╵                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/drawing_utilities"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:9:31:
      9 │ import { apply_gradient } from '../build/drawing_utilities'
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/configuration"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:10:50:
      10 │ import { Configuration, AllowedToolboxItem } from '../build/configuration';
         ╵                                                   ~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/html_builder"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:11:28:
      11 │ import { HTMLBuilder } from '../build/html_builder';
         ╵                             ~~~~~~~~~~~~~~~~~~~~~~~

dist-electron/main.js  571.00 kB │ gzip: 169.08 kB
built in 33165ms.

X [ERROR] Could not resolve "../build/annotation"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:5:33:
      5 │ import { ULabelAnnotation } from '../build/annotation';
        ╵                                  ~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/subtask"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:6:30:
      6 │ import { ULabelSubtask } from '../build/subtask';
        ╵                               ~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/geometric_utils"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:7:31:
      7 │ import { GeometricUtils } from '../build/geometric_utils';
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/annotation_operators"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:8:42:
      8 │ import { get_annotation_confidence } from '../build/annotation_operators';
        ╵                                           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/drawing_utilities"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:9:31:
      9 │ import { apply_gradient } from '../build/drawing_utilities'
        ╵                                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/configuration"

    node_modules/.store/ulabel@0.6.5/node_modules/ulabel/src/index.js:10:50:
      10 │ import { Configuration, AllowedToolboxItem } from '../build/configuration';
         ╵                                                   ~~~~~~~~~~~~~~~~~~~~~~~~

X [ERROR] Could not resolve "../build/html_builder"

### PR DESCRIPTION
**Pull request by TrevorBurgoyne**
# Fix npm build

## Description
- Previous work had simplified the build process to exclude the use of a published `build/` folder with the converted ts code, however the package entrypoints in `package.json` were not ever updated to reflect this change. This was causing import errors as `index.js` was still the main entrypoint and it tried to import from non-existent `build` files (see #123).
- Closes #123 by updating the package entrypoint to be `dist/ulabel.js` 

## PR Checklist

- [x] Merged latest main
- [x] Version number in `package.json` has been bumped since last release
- [x] Version numbers match between package `package.json` and `src/version.js`
- [x] Ran `npm install` and `npm run build` AFTER bumping the version number
- [x] Updated documentation if necessary (currently just in `api_spec.md`)
- [x] Added changes to `changelog.md`

## Breaking API Changes
no

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,13 +2,13 @@
   "name": "ulabel",
   "description": "An image annotation tool.",
   "version": "0.6.10",
-  "main": "index.js",
-  "module": "src/index.js",
+  "main": "dist/ulabel.js",
+  "module": "dist/ulabel.js",
```
Comment (TrevorBurgoyne) [inline package.json:6]: does it make sense to make one/both of these the minified version? tbh I'm not sure if there's any difference btwn `main` and `module`

File: dist/ulabel.min.js.LICENSE.txt [hunk 2]
```diff
@@ -20,5 +20,7 @@
  * Released under the MIT license
  * https://jquery.org/license
  *
- * Date: 2022-12-20T21:28Z
+ * Date: 2021-03-02T17:08Z
  */
+
+/*! For license information please see ulabel.js.LICENSE.txt */
```
Comment (csolbs24) [inline dist/ulabel.min.js.LICENSE.txt:26]: Same here. Not sure what's up with this?

File: package-lock.json
Comment (csolbs24) [inline package-lock.json:1]: Looks like you're reverting a lot of the package versions. Is this intentional?

File: dist/ulabel.js.LICENSE.txt [hunk 3]
```diff
@@ -20,5 +20,7 @@
  * Released under the MIT license
  * https://jquery.org/license
  *
- * Date: 2022-12-20T21:28Z
+ * Date: 2021-03-02T17:08Z
  */
+
+/*! For license information please see ulabel.js.LICENSE.txt */
```
Comment (csolbs24) [inline dist/ulabel.js.LICENSE.txt:26]: Not sure exactly why the license seems like its being reverted?

**Review by csolbs24 — COMMENTED** [review]
Nice find on package.json's main field being incorrect. Just looks like there's some weird stuff going on with a lot of stuff being reverted.

File: package.json (hunk 1 above)
Comment (csolbs24) [reply package.json:6]: No, I think the un-minified version should remain the default. I personally am not super familiar with node packages work tbh but this should explain the difference between main and module https://stackoverflow.com/questions/42708484/what-is-the-module-package-json-field-for

**Review by csolbs24 — COMMENTED** [review]
Please bump the version and add changes to the changelog. (Don't worry about the previous version's changes not being added to the changelog. I added it to my currently open pr)

**Comment by TrevorBurgoyne** [comment]
> Nice find on package.json's main field being incorrect. Just looks like there's some weird stuff going on with a lot of stuff being reverted.

yeah idk what was going on, my local builds were wack so I've fallen back to the 6.10.0 files

**Review by csolbs24 — APPROVED** [review]
LGTM 👍

**Pull request merged**